app/client/src/tests/unit/sidemenu.test.js

```diff
@@ -0,0 +1,149 @@
+import React from 'react';
```

Comment:
the previous file is name App.test.js, but this one is sidemenu.test.js.

Do we want files to start with a capital letter or lower case? Whats the defined best practice for this?

Reply:
I would think either file name or name of the React component. 
So in this case, either sideMenu or SideMenu.

Reply:
Double check how the capstone team is creating/naming files and lets stick to the one standard.